app/views/lookup_keys/_fields.html.erb

```diff
@@ -12,30 +12,21 @@
   </fieldset>
   <fieldset>
     <h2><%= _("Default behavior") %></h2>
+    <%= text_f(f, :puppet_default_value, :size => "col-md-8", :disabled => true) if is_param %>
```

Comment:
Should be as part of: http://projects.theforeman.org/issues/15503 and not this pr.